### PR TITLE
Playwright tests and minor design

### PR DESCRIPTION
**Pull request by shech2**
check this out.

### CODE REVIEW

**Comment by idoRosen25** [comment]
Not sure I understand what is done here. Seems it also includes my invitation email changes when it shouldn't have them.
Rebase and make sure only the new changes exist. Otherwise mark as draft until I merge my changes

**Comment by shech2** [comment]
> Not sure I understand what is done here. Seems it also includes my invitation email changes when it shouldn't have them. Rebase and make sure only the new changes exist. Otherwise mark as draft until I merge my changes

i added a plus button on header and put invite page inside the layout and added tests

**Comment by shech2** [comment]
There is a problem with the env... but i dont know why

File: package.json [hunk 1]
```diff
@@ -12,6 +12,7 @@
     "prisma:generate": "npx prisma generate",
     "prisma:validate": "npx prisma validate",
     "migrate:postgres": "prisma migrate dev",
+    "migrate:dev": "dotenv -e .env.local -- npx prisma migrate dev",
```
Comment (idoRosen25) [inline package.json:15]: Why is this needed?

File: package.json [hunk 2]
```diff
@@ -51,6 +54,8 @@
     "@types/react": "^18",
     "@types/react-dom": "^18",
     "autoprefixer": "^10.0.1",
+    "dotenv": "^16.4.1",
```
Comment (idoRosen25) [inline package.json:57]: Why do we need this?

File: tests-examples/demo-todo-app.spec.ts [hunk 3]
```diff
@@ -0,0 +1,437 @@
+import { test, expect, type Page } from '@playwright/test';
+
```
Comment (idoRosen25) [inline tests-examples/demo-todo-app.spec.ts:2]: Is this file needed? Seems like an example and not a true test

**Review by idoRosen25 — CHANGES_REQUESTED** [review]
Seems fine on the whole. Some points I'm not sure what they mean

File: package.json (hunk 1 above)
Comment (shech2) [reply package.json:15]: I tried to load the env to run the workflows , it doesn't work , we need to add secrets to GitHub in order for that to work

File: package.json (hunk 2 above)
Comment (shech2) [reply package.json:57]: SAO

File: tests-examples/demo-todo-app.spec.ts (hunk 3 above)
Comment (shech2) [reply tests-examples/demo-todo-app.spec.ts:2]: Nope i will delete it

File: package.json (hunk 1 above)
Comment (shech2) [reply package.json]: Removed this line

**Comment by shech2** [comment]
Mark as draft too many commits, i will do it cleaner later and delete this pull..
